Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/model/Match.java

```diff
@@ -0,0 +1,27 @@
+package org.cqframework.cql.cql2elm.model;
+
+public enum Match {
```

Comment:
This is an enum of Match _types_, and so all the code in general would read better if this was called `MatchType`